deadspace/code/marker/markernet/marker_eye.dm

```diff
@@ -185,6 +185,42 @@ GLOBAL_LIST_EMPTY(markers_signals)
 
 	qdel(src)
 
+/mob/camera/marker_signal/verb/switch_necroqueue()
+	set name = "Join/Leave Necroqueue"
+	set category = "Necromorph"
+
+	if(!marker.active)
+		to_chat(src, span_notice("Marker is not active yet!"))
+		return
+	if(src in marker.necroqueue)
+		to_chat(src, span_notice("You are now in the necroqueue. When a necromorph vessel is available, you will be automatically placed in control of it. You can still manually posess necromorphs."))
+		marker.necroqueue += src
+	else
+		to_chat(src, span_notice("You have left the necroqueue."))
+		marker.necroqueue -= src
+
+/mob/camera/marker_signal/verb/jump_to_maker()
+	set name = "Jump to Marker"
+	set category = "Necromorph"
+
+	forceMove(get_turf(marker))
```

Comment:
might want to use `abstract_move()`

Reply:
`forceMove()` uses `abstract_move()` already (doesn't break anything so far)
```
/mob/camera/marker_signal/forceMove(atom/destination)
	abstract_move(destination) // move like the wind
	return TRUE
```